test/langtools/tools/javac/T8314423.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * @test  /nodynamiccopyright/
+ * @bug 8314423
+ * @summary Multiple patterns without unnamed variables
```

Comment:
Maybe use `@enablePreview` instead of `--enable-preview`?

Reply:
I would like to avoid the `@enablePreview` to be applied to both `@compile` commands. In the first I test that it will raise the error by checking the preview feature, and in the second that it will compile successfully. Isn't `@enablePreview` going to be applied to both?

Reply:
Ah, sorry, I missed that.